modules/manage/pages/monitor/monitor-intro.adoc

```diff
@@ -49,4 +49,6 @@ Statistics for the Index Service can be managed by means of Couchbase Web Consol
 
 The monitoring of statistics related to the Query Service is described in xref:manage:monitor/monitoring-n1ql-query.adoc[Monitor Queries].
 
+It's also possible to set up a Prometheus monitor to consume metrics data from a Couchbase cluster. You can find an introduction on how to do this in xref:monitor/set-up-prometheus-for-monitoring.adoc[]
```

Comment:
I suggest making a separate subsection on this page called 'Monitoring with External Prometheus' and putting these 2 sentences under that subsection.

Reply:
Done